[2025-03-18T23:46:46.048+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-18T23:46:46.057+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: sevdesk_contact_pipeline.get_contacts scheduled__2025-03-18T22:00:00+00:00 [queued]>
[2025-03-18T23:46:46.061+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: sevdesk_contact_pipeline.get_contacts scheduled__2025-03-18T22:00:00+00:00 [queued]>
[2025-03-18T23:46:46.062+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 3
[2025-03-18T23:46:46.153+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): get_contacts> on 2025-03-18 22:00:00+00:00
[2025-03-18T23:46:46.159+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=76) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-03-18T23:46:46.160+0000] {standard_task_runner.py:72} INFO - Started process 77 to run task
[2025-03-18T23:46:46.160+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'sevdesk_contact_pipeline', 'get_contacts', 'scheduled__2025-03-18T22:00:00+00:00', '--job-id', '347', '--raw', '--subdir', 'DAGS_FOLDER/sevdesk_contact_pipeline.py', '--cfg-path', '/tmp/tmp9tobw1ip']
[2025-03-18T23:46:46.161+0000] {standard_task_runner.py:105} INFO - Job 347: Subtask get_contacts
[2025-03-18T23:46:46.197+0000] {task_command.py:467} INFO - Running <TaskInstance: sevdesk_contact_pipeline.get_contacts scheduled__2025-03-18T22:00:00+00:00 [running]> on host e54e5582f32f
[2025-03-18T23:46:46.238+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='sevdesk_contact_pipeline' AIRFLOW_CTX_TASK_ID='get_contacts' AIRFLOW_CTX_EXECUTION_DATE='2025-03-18T22:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-03-18T22:00:00+00:00'
[2025-03-18T23:46:46.238+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-03-18T23:46:46.239+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-03-18T23:46:46.239+0000] {logging_mixin.py:190} INFO - Current task name:get_contacts state:running start_date:2025-03-18 23:46:46.057653+00:00
[2025-03-18T23:46:46.239+0000] {logging_mixin.py:190} INFO - Dag name:sevdesk_contact_pipeline and current dag run status:running
[2025-03-18T23:46:46.239+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-03-18T23:46:46.470+0000] {logging_mixin.py:190} INFO - âœ… Contact already exists: 99245309
[2025-03-18T23:46:46.471+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-03-18T23:46:46.481+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-03-18T23:46:46.481+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=sevdesk_contact_pipeline, task_id=get_contacts, run_id=scheduled__2025-03-18T22:00:00+00:00, execution_date=20250318T220000, start_date=20250318T234646, end_date=20250318T234646
[2025-03-18T23:46:46.501+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2025-03-18T23:46:46.501+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-03-18T23:46:46.502+0000] {logging_mixin.py:190} INFO - Dag name:sevdesk_contact_pipeline queued_at:2025-03-18 23:46:43.562850+00:00
[2025-03-18T23:46:46.502+0000] {logging_mixin.py:190} INFO - Task hostname:e54e5582f32f operator:PythonOperator
[2025-03-18T23:46:46.508+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-18T23:46:46.527+0000] {taskinstance.py:3901} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-03-18T23:46:46.528+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
